pyquil/tests/test_quil.py

```diff
@@ -38,17 +40,33 @@ def test_gate():
 
 
 def test_defgate():
-    dg = DefGate("TEST", np.array([[0 + 0.5j, 0.5],
-                                   [0.5, 0 - 0.5j]]))
-    assert dg.out() == "DEFGATE TEST:\n    0.0+0.5i, 0.5+0.0i\n    0.5+0.0i, 0.0-0.5i\n"
+    dg = DefGate("TEST", np.array([[sqrt(0.5), sqrt(0.5)],
+                                   [-sqrt(0.5) * 1j, sqrt(0.5) * 1j]]))
+    assert dg.out() == "DEFGATE TEST:" \
+                       "\n    0.70710678118654757+0.0i, 0.70710678118654757+0.0i"\
```

Comment:
Am a little skittish about the stability of string comparisons of floats as options like `np.isclose` aren't available.  Can you change this test example to be an example without sqrt computation.  For example [[1,0],[0,1]] would be fine.